README.md

```diff
@@ -48,161 +146,276 @@ To run the demo code, follow these steps:
 
 ## API
 
-MySqlConnection objects provide seven methods.
-One way to use the returned promise is to chain calls to `then` and `catch`.
-Another is to use `async` and `await`.
+### MySqlEasier
+
+This is a global object that represents the common use cases.
+It is intended to be a singleton in a node application to allow various modules to share a single MySQL connection pool.
+
+```js
+const mySqlEasier = require('mysql-easier');
+```
+
+#### `configure`
+Creates the global MySqlPool using the provided configuration.
+
+```js
+mySqlEasier.configure({
+  host: 'localhost',
+  user: 'myuser',
+  password: 'super-secret'
+});
+```
+
+#### `createConnection`
+Creates a new MySqlConnection using a non-pooled connection with the provided configuration.
+
+This method should only be used if you intentionally do not wish to use pooled connections.
+
+When you are done with the connection, you should call `myConnection.done()` or `myConnection.destroy()`.
```

Comment:
What's the difference between done and destroy?

Reply:
Destroy completely stops the connection.  Once a connection has been destroyed it cannot be reused.  Generally, you want to use `done`.  I notated that in the documentation for those methods, but not here.  Perhaps I should remove the `destroy` option here to avoid confusion.

Reply:
That sounds good. That way you'd just be demonstrating the recommended usage while still documenting less common usage later.